### PR TITLE
Fix healthCheckUrl method

### PR DESCRIPTION
**Pull request by Arhius**

### CODE REVIEW

File: src/messages.js [hunk 1]
```diff
@@ -59,7 +59,7 @@ const messages = {
   onboardingSuccessful: '🎉 You are now successfully onboarded !!! 🎉',
   alreadyOnboarded: (role) => `✅ Onboarded as ${role}`,
   networkSelected: (network) => `Network: ${network}`,
-  healthCheckUrl: `After starting your node, you can check its health by going to Explorer: https://explorer.ambrosus.com/`,
+  healthCheckUrl: () => `After starting your node, you can check its health by going to Explorer: https://explorer.ambrosus.com/`,
```
Comment (Anze1m) [inline src/messages.js]: Why no parameter?

Comment (Arhius) [reply src/messages.js]: From previous PR. @seagiv changed with no parameters. Just to print explorer url.

**Comment by krzysztof-jelski** [comment]
It improves from the broken state, so I approve it. Just to make our community's life easier.

But:
1) I'm not convinced directing users to `https://explorer.ambrosus.com/` is better than to `health` on their node. I understand that the problem we're trying to solve is: user goes to `http://my-node.com/health` and sees an empty page and does not know what it means. I think better of solving the problem would be: direct them to `/nodeinfo`, instruct on what to expect, direct them to `https://explorer.ambrosus.com/nodes/0x0000nodeaddress`
2) all URLs output by NOP are in yellow chalk, so I propose that one should be as well.

File: src/messages.js (hunk 1 above)
Comment (Anze1m) [reply src/messages.js]: But why print explorer URL, while this is `healthCheckUrl` message, used in `healthCheckUrlDialog`, which is used to display node's healthCheck endpoint URL.

**Comment by Arhius** [comment]
I don't know why it was changed to explorer url. I will ask @seagiv about his previous task

**Comment by Arhius** [comment]
I will ask Alina to make task for this issue. This is just hotfix for community

**Comment by seagiv** [comment]
Our management told me to change URL to explorer, you should ask Valera about,as for /health the problem was exactly what Krzysztof said, users do not understand empty page and think it is an error.

**Pull request merged**